SU-HSDO/stanford_mrc
# MRCD8-249 fix news and events spacing issue and add margin mixin to patterns

# READY FOR REVIEW

# Summary
- Fix margin issue on Event and News cards (list views)
- Added margin bottoms to cards; added margin mixins to patterns for consistency

# Needed By (Date)
- End of sprint (3/1)

# Urgency
- N/A

# Steps to Test
1. Pull this branch
2. Verify that the list cards on the /events and /news landing pages are not touching one another anymore

# Affected Projects or Products
- stanford_mrc
- MRC

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-249

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
